### PR TITLE
fix #41496 coinhub.pw

### LINKED ISSUE #41496: coinhub.pw
### Issue URL (Anti Adblock Script)
[https://coinhub.pw/?shortlink=Z5HY4JKEA8B](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcoinhub.pw%2F%3Fshortlink%3DZ5HY4JKEA8B)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard German,<br>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/41496

### CODE REVIEW

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -189,6 +189,7 @@ expressilustrowany.pl#%#AG_abortInlineScript(/googlefc/, 'atob');
 realadmin.ru##.wrap_article > .money
 pureinfotech.com##body > div[style*="z-index: 9999"][style*="cursor: pointer;"]
 it-actual.ru##.blok[style="text-align: center;"]
+coinhub.pw#?#body > center:contains(Turn off your adblocker)
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/antiadblock.txt]: ext-CSS

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/antiadblock.txt]: what is wrong?

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -18,25 +18,25 @@
...
...win.com,softonic.com,softwaregiveaway.xyz,sol.dk,splinternews.com,sport-tv-guide.live,ssulwar.com,stars24.cz,streamapi.xyz,streamcloud.eu,surfsees.com,svapo.it,syreop.com,tagblatt.ch,teachios.com,techhung.com,techkreative.xyz,techperiod.com,techwidetoday.site,tecnofullpc.com,tecnotutoshd.net,telecinco.es,telegraph.co.uk,tetewlink.me,thaivisa.com,thechive.com,thefilmbay.com,theonion.com,thepiratefilmestorrent.com,theroot.com,thesurfersview.com,thetakeout.com,today.it,tokenfly.pw,topserialy.to,topspeed.sk,torrentdosfilmeshd.net,trecetv.es,trendenciashombre.com,truyenaudiocv.com,tsa-algerie.com,tubitv.com,tugaflix.com,turkanime.tv,tv.kakao.com,tvple.com,tvserial.it,tvsportslive.stream,unikampus.net,uploaded.net,upost.info,utimetableresult.in,vacation-et.work,vcstream.to,vergol.com,veuclips.com,vev.io,videostreamlet.net,vidoza.net,viki.com,vipshare.co,viuclips.net,vmf.edge-apps.net,vpnssh.net,vtvgiaitri.vn,watchseries1.cc,weatherx.co.in,web1907.tistory.com,webzook.net,wikiall.org,win10.vn,windows7forum.pl,winhelp.us,wordgames.com,worldfreeware.com,wpteq.org,xatakandroid.com,xboxdvr.com,xkorean.biz,yafud.pl,yesmoviesapp.info,yuncomics.com,zxctunnel.com#%#//scriptlet("prevent-fab-3.2.0")
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt]: this rule causes adblock detection with disabled filters.
This rule also required `#%#//scriptlet('abort-current-inline-script', 'adBlockNotDetected', '/FuckAdBlock/')`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
I will make a new pr because there is a lot of changes.

**Pull request closed without merging**